bootstrap/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:buster
+COPY setup.py /
+COPY startup.json.default /
+COPY main.py /
```

Comment:
you know that you can do `COPY setup.py main.py startup.json.default /` right ?

Reply:
I do now!